packages/components/src/visually-hidden/index.js

```diff
@@ -19,4 +19,26 @@ function VisuallyHidden( { as = 'div', className, ...props } ) {
 		...props,
 	} );
 }
+
+/**
+ * VisibilityHelper component to simplify common use-case of
+ * conditionally hiding components.
+ */
+export function VisibilityHelper( {
```

Comment:
It's not clear to me why we need this component? It feels like something that should be done by the caller in general?

Reply:
Also, why a new component, why not just a prop for VisuallyHidden

Reply:
A prop was my first thought but I couldn't come up with a good name, e.g. all of these feels awkward: `<VisuallyHidden isHidden={true} />`, `<VisuallyHiddenMaybe isHidden={true} />`